atomicapp/cli/main.py

```diff
@@ -227,25 +227,29 @@ def create_parser(self):
             dest="namespace",
             help=('The namespace to use in the target provider'))
         globals_parser.add_argument(
-            "--providertlsverify",
+            "--provider-tlsverify",
             dest="providertlsverify",
```

Comment:
Need to change this line to add a dash. I suspect that is why you are seeing the error you are seeing.